source/jormungandr/tests/direct_path_asgard_integration_tests.py

```diff
@@ -182,4 +216,28 @@ def test_journey_with_bike_direct_path(self):
         assert response['journeys'][2]['duration'] == 2000
         assert not response['journeys'][2]['sections'][0].get('cycle_lane_length')
 
-        assert not response.get('feed_publishers')
+
+@dataset(
+    {
+        'main_routing_test': {
+            'scenario': 'distributed',
+            'instance_config': {'street_network': MOCKED_ASGARD_CONF_WITH_BAD_RESPONSE},
+        }
+    }
+)
+class TestAsgardDirectPath(AbstractTestFixture):
+    def test_crowfly_replaces_section_if_street_network_failed(self):
+        """
+        Topic: This case arrives when the street network computation has failed.
+               In this case, we replace the lost street network section by a crowfly, like in New Default
+        """
+        response, status = self.query_region(journey_basic_query, check=False)
+        assert status == 200
+        check_journeys(response)
+
+        assert len(response['journeys']) == 1
+
+        assert len(response['journeys'][0]['sections']) == 3
+        assert response['journeys'][0]['sections'][0]['type'] == 'crow_fly'  # replace the street network
```

Comment:
to be even more expressive and leave a hint if the test fails, you can decorate your assertion with an error message :
```suggestion
        assert response['journeys'][0]['sections'][0]['type'] == 'crow_fly', "A crow fly should replace the street network"
```

But that's only if you want to be fancy :)